src/testcafe.js

```diff
@@ -42,10 +45,18 @@ export default class TestCafe {
     }
 
     createRunner () {
-        return new Runner(this.proxy, this.browserConnectionGateway);
+        var newRunner = new Runner(this.proxy, this.browserConnectionGateway);
+
+        this.runners.push(newRunner);
+        promisifyEvent(newRunner, 'stopped').then(() => remove(this.runners, newRunner));
+
+        return newRunner;
     }
 
-    close () {
+    async close () {
+        await Promise.all(this.runners.slice().map(runner => runner.stop()));
```

Comment:
Well, the best way will be to stop by gracefully shutting everything down. I think we should start from other point. For `testcafe-studio` we will need ability to `cancel` task Promise which is returned by the `runner.run()`. So we need to extend this Promise with the `cancel()` method which will behave like if we have fatal run error: aborts tasks and gracefully closes remaining unused connection. So, if we we'll implement it, we can have `pendingTaskPromises` array in `runner`, so testcafe will just `cancel` all such Promises in all active tasks and then just `close` proxy. \cc @AndreyBelym